PalasoUIWindowsForms.GeckoFxWebBrowserAdapter/WebThumbnailViewer.cs

```diff
@@ -133,7 +133,16 @@ public void LoadItems(IEnumerable<string> pathList)
 				sb.AppendLine("<div class='imageWrap' id='" + htmlPath
 					+ "'><div class='imageWrapRel'><div class='imageWrapOuter'><div class='imageWrapMid'><div class='imageWrapInner'>");
 				// the data-echo attribute is part of lazy loading for very long lists (see below).
-				sb.AppendLine("<img class='image' src='' data-echo='file://" + new Uri(GetThumbnailFile(path)).AbsolutePath + "'>");
+
+				if (PlatformUtilities.Platform.IsWindows)
+				{
+					sb.AppendLine("<img class='image' src='' data-echo='file://" + htmlPath + "'>");
+				}
+				else
+				{
+					sb.AppendLine("<img class='image' src='' data-echo='file://" + new Uri(GetThumbnailFile(path)).AbsolutePath + "'>");
```

Comment:
It would be, at best, tricky to apply that idea in this case, because the image server is Bloom code, but this AOR code is shared by other libpalaso clients.